lib/active_merchant/billing/gateways/litle.rb

```diff
@@ -197,7 +197,10 @@ def add_payment_method(doc, payment_method, options)
         if payment_method.is_a?(String)
           doc.token do
             doc.litleToken(payment_method)
-            doc.expDate(options[:exp_date]) if options[:exp_date]
+            if options[:exp_month] && options[:exp_year]
+              expiration_date = "#{format(options[:exp_month], :two_digits)}#{format(options[:exp_year], :two_digits)}"
```

Comment:
Maybe refactor the existing `expiration_date` method so that it can be used here too?